openid-4-verifiable-presentations-1_0.md

```diff
@@ -549,11 +555,273 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
+
+`credential_signing_alg_values`:
+: OPTIONAL. An array of strings that specifies
+the allowed algorithms for signing the SD-JWT (i.e., only the issuer-signed part).
+
+`kbjwt_signing_alg_values`:
+: OPTIONAL. An array of strings that specifies the
+allowed algorithms for signing the Key-Binding JWT (KB-JWT) (i.e., the part signed
+by the holder).
+
+`sd_alg_values`:
+: OPTIONAL. An array of strings that specifies the allowed hash algorithms for
+digests in the SD-JWT and KB-JWT.
+
+### Format `mso_mdoc` {#format_mso_mdoc}
+
+TBD
+
+### Format `jwt_vp*`
+
+TBD
+
+
+## Claims Path Pointer {#claims_path_pointer}
+
+A claims path pointer is a pointer into the JSON structure of the Verifiable
+Credential, identifying one or more claims. A claims path pointer MUST be a
+non-empty array of strings and non-negative integers. A `*` string value indicates
+that all elements of the currently selected array(s) are to be selected; any other
+string indicates that the respective key is to be selected; and a non-negative
+integer indicates that the respective index in an array is to be selected. The path
+is formed as follows:
+
+  - Start with an empty array.
+  - To address a particular claim within an object, append the key (claim name)
+    to the array.
+  - To address an element within an array, append the index to the array (as a
+    non-negative, 0-based integer).
+  - To address all elements within an array, append the string `*` to the array.
+
+Verifiers MUST NOT point to the same claim more than once in a single query.
+Wallets SHOULD ignore such duplicate claim queries.
+
+### Example
+
+The following shows a non-normative, simplified example of a Credential:
+
+```json
+{
+  "name": "Arthur Dent",
+  "address": {
+    "street_address": "42 Market Street",
+    "locality": "Milliways",
+    "postal_code": "12345"
+  },
+  "degrees": [
+    {
+      "type": "Bachelor of Science",
+      "university": "University of Betelgeuse"
+    },
+    {
+      "type": "Master of Science",
+      "university": "University of Betelgeuse"
+    }
+  ],
+  "nationalities": ["British", "Betelgeusian"]
+}
+```
+
+The following shows examples of claims path queries and the respective selected
+claims:
+
+- `["name"]`: The claim `name` with the value `Arthur Dent` is selected.
+- `["address"]`: The claim `address` with its sub-claims as the value is
+  selected.
+- `["address", "street_address"]`: The claim `street_address` with the value `42
+  Market Street` is selected.
+- `["degrees", "*", "type"]`: All `type` claims in the `degrees` array are
+  selected.
+- `["nationalities", 1]`: The second nationality is selected.
+- `[]` (empty array): The entire Credential is selected.
+
+### Processing
+
+In detail, the array is processed by the Wallet from left to right as follows:
+
+ 1. Select the root element of the credential, i.e., the top-level JSON object.
+ 2. Process the query components from left to right:
```

Comment:
The term "query components" is never mentioned before this

Reply:
Thanks, I'll change this to make clear what is meant.